ROVU/Core/src/utility/Main.java

```diff
@@ -0,0 +1,112 @@
+package utility;
+import java.awt.Color;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import controller.RobotController;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Rectangle2D;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Screen;
+import javafx.stage.Stage;
+import project.AbstractSimulatorMonitor;
+import project.Point;
+import robot.RobotHandler;
+import simbad.sim.AbstractWall;
+import simbad.sim.Boundary;
+import simbad.sim.EnvironmentDescription;
+import simbad.sim.HorizontalBoundary;
+import simbad.sim.HorizontalWall;
+import simbad.sim.VerticalBoundary;
+import simbad.sim.VerticalWall;
+import view.ROVUView;
+
+@SuppressWarnings("unused")
+public class Main extends Application{
+	
+	static RobotHandler robot1;
+	static RobotHandler robot2;
+	static RobotHandler robot3;
+	static RobotHandler robot4;
+	
+	 Stage stage = new Stage();
+	    public static Scene scene;
```

Comment:
You should use `static` variables with caution. I suggest you remove `static`, and make them `private` as well.